README.md

```diff
@@ -3,71 +3,46 @@
 [![DOI](https://zenodo.org/badge/632689876.svg)](https://zenodo.org/badge/latestdoi/632689876)
 [![Chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://acquire-imaging.zulipchat.com/)
 
-# acquire-imaging
+# Acquire
 
-This python package provides a multi-camera video streaming library focusing
-on performant microscopy.
+### Installation
```

Comment:
```suggestion
### Installation
```

Lets just get rid of the heading. Alternatively could add an `About` heading before line 14.

Reply:
I think it looks better with Installation deleted than with having both Installation and About headings